### PR TITLE
ignore empty fault-failure codes

### PR DESCRIPTION
**Pull request by ipa-hsd**
some bugs (e.g. NOT-A-BUG) may not have any fault/failure code assigned. Skips such bugs instead of throwing exception

### CODE REVIEW

File: scripts/gdocs_reader.py [hunk 1]
```diff
@@ -101,27 +101,35 @@ def refactor_file(filename: str,
...
     if not failure_sys:
-        raise ValueError(filename, "No system failure codes assigned to the bug.")
+        print(f"{filename}: No system failure codes assigned to the bug")
     else:
         failure_sys_list = failure_sys.upper().splitlines()
 
     failure_list: t.Optional[t.List[str]]
-    failure_list = failure_sw_list + failure_sys_list
+    failure_list = None
+    if failure_sw_list != None and failure_sys_list != None:
```
Comment (gavanderhoorn) [inline scripts/gdocs_reader.py:128]:
```suggestion
    if failure_sw_list and failure_sys_list:
```
?

File: scripts/gdocs_reader.py [hunk 2]
```diff
@@ -101,27 +101,35 @@ def refactor_file(filename: str,
...
+        failure_list = failure_sw_list + failure_sys_list
+
+    if fault_list == None and failure_list == None:
```
Comment (gavanderhoorn) [inline scripts/gdocs_reader.py:131]:
```suggestion
    if not fault_list and not failure_list:
```

**Comment by ChrisTimperley** [comment]
Is this good to merge, @ipa-hsd, @gavanderhoorn?

**Comment by ipa-hsd** [comment]
@ChrisTimperley I have incorporated suggestions from @gavanderhoorn. So from my side, yes.

**Pull request merged**